packages/editor/src/browser/editor-navigation-contribution.ts

```diff
@@ -91,6 +99,25 @@ export class EditorNavigationContribution implements Disposable, FrontendApplica
             execute: () => this.toggleWordWrap(),
             isEnabled: () => true,
         });
+        this.commandRegistry.registerHandler(EditorCommands.REOPEN_CLOSED_EDITOR.id, {
+            execute: () => this.reopenLastClosedEditor()
+        });
+    }
+
+    async reopenLastClosedEditor(): Promise<void> {
```

Comment:
I believe it should be `protected`?
I do not expect callers to inject the contribution and call this method. If they wish to reopen a closed editor they can execute the command itself:

```ts
this.commandRegistry.executeCommand(EditorCommands.REOPEN_CLOSED_EDITOR.id);
```